app/policies/course_policy.rb

```diff
@@ -39,6 +39,7 @@ def send_vacancies_updated_notification?
 
   alias_method :details?, :show?
   alias_method :update?, :show?
+  alias_method :edit?, :show? # is this the correct permission?
```

Comment:
For the moment it is fine as the basic permission available is 
- if you can see it then you can edit it